rootston/output.c

```diff
@@ -841,6 +842,17 @@ void handle_new_output(struct wl_listener *listener, void *data) {
 		roots_config_get_output(config, wlr_output);
 	if (output_config) {
 		if (output_config->enable) {
+			struct roots_output_mode_config *mode_config;
+
+			if (wlr_output_is_drm(wlr_output)) {
+				wl_list_for_each(mode_config, &output_config->modes, link) {
+					wlr_drm_connector_add_mode(wlr_output, &mode_config->info);
+				}
+			} else {
+				if (wl_list_length(&output_config->modes)) {
```

Comment:
If you're going to update this PR, here's a nitpick: it's better to use `wl_list_empty` here instead of `wl_list_length` because it is constant time.

Reply:
Ping?

Reply:
Seems github hates me. Seems the PR updated now correctly.